llvm/test/Verifier/denormal-fp-math.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: not llvm-as %s -o /dev/null 2>&1 | FileCheck %s
+
+define float @test_denormal_fp_math_valid() "denormal-fp-math"="ieee,ieee" {
+  ret float 1.0
+}
```

Comment:
This doesn't test anything. Don't really need it, but you need --implict-check-not=error or something to make this check there's no error